variables.tf

```diff
@@ -22,6 +22,7 @@ variable "project_id" {
 variable "prefix" {
   description = "Prefix used to generate the bucket name."
   type        = string
+  default     = ""
```

Comment:
@pkatsovich Could you remove this? While I agree with it, this is a breaking change which I'd like to avoid.

Reply:
Thanks for the quick review, @morgante  . I've removed this change.